src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1277,6 +1294,18 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 				}
 
 				if (this._processManager.processState === ProcessState.KilledDuringLaunch) {
+				
+					// https://github.com/microsoft/vscode/issues/132578 
+					if (this._exitCode === 267){
```

Comment:
Could you repro? I'm worried this won't actually fix the problem and you would actually need to catch the exception inside `terminalProcess.ts`, also this should only be handled on Windows.

Reply:
You're right I underestimated the cases to handle. I'll leave this one.